src/rockstor/fs/tests/test_btrfs.py

```diff
@@ -1,5 +1,5 @@
 """
-Copyright (c) 2012-2017 RockStor, Inc. <http://rockstor.com>
+Copyright (c) 2012-2017 RockStor, Inc. <https://rockstor.com>
```

Comment:
no functional value add, but maybe also adjust the copyright years: `2012-2023`

Reply:
Do you want to pop in the changes then - I can then merge this working pr and work on the other 30 unit test mock transitions. :)

Reply:
i.e. modify your suggestion here so I can commit it.

Reply:
```suggestion
Copyright (c) 2012-2023 RockStor, Inc. <https://rockstor.com>
```

Reply:
Thanks @Hooverdan96 These are important - as well as the https as I'm trying to remove as many warnings as possible form the editor so it become useful again and it always flags all http as an issue !!